Paco-iOS/Podfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment this line to define a global platform for your project
```

Comment:
Well, I will keep in mind. At the moment, I'm working hard to improve application performance on the iPhone 6 and iPhone 6 Plus.